app/javascript/packs/shared.js

```diff
@@ -10,7 +10,6 @@ import '../src/konami_code'
 
 import "controllers"
 
-require("@rails/ujs").start()
```

Comment:
is this definitely no longer required?

Reply:
readded by [bfa9fd6](https://github.com/EdinburghUniversityTheatreCompany/black_lightning/pull/294/commits/bfa9fd615dcdcd6ba93133150fa52346f37c6ff4)